app/lib/services/handle_submission_for_store.rb

```diff
@@ -67,7 +67,7 @@ def raw_application_data
       # Cutting out confirmation keys since that is not application related data
       # Though I recognise that this means that even though this is meant to be raw
       # it still has a small layer of processing
-      store.except("generated_confirmation_code")
+      store.except("generated_confirmation_code", "choose_your_npq")
```

Comment:
I'm not sure why this would be exempt from storage, this is still an answer the user entered and the goal of `raw_application_data` is to store a raw recording of exactly what happened in the users application, which would include this.

Reply:
I guess this makes a difference depending on the other comment.